src/components/trace/hat-script-graph.ts

```diff
@@ -143,77 +109,62 @@ class HatScriptGraph extends LitElement {
     graphStart = false
   ) {
     const trace = this.trace.trace[path] as ChooseActionTraceStep[] | undefined;
```

Comment:
`trace = this.trace.trace[path]?.[0]`... and so on?
It's only the first one that's ever used (right?), and it avoids the confusing `map`.

Reply:
The `choose` could be run in a `repeat`, and then could have multiple paths taken, this way we track them all.